confero-front/src/components/theme.ts

```diff
@@ -0,0 +1,5 @@
+import { createMuiTheme } from "@material-ui/core/styles";
+
+const theme = createMuiTheme({});
```

Comment:
this is where we could customize the [Material UI theme](https://material-ui.com/customization/theming/)